### ISSUE TITLE
My problem

### ISSUE THREAD

**Issue by nekenet** [JerryWm/Stratum-Proxy-XMR#15]
Hey. You have a very good proxy! I liked a different difficulty for miners. But there are several questions.
Why does the hash-rate jump? I have 17 miners, and with a straight line on the pool, it shows a hash rate of 2500 h / s. But when using a proxy, the hash-rate jumps between 800 and 3500 h / s. I understand that you need to wipe the hash-rate for 24 hours, but it's lower than when you dig out a 2300-2100 h / s.
What can I do to solve the problem?

**Comment by 2010phenix** [comment]
to big diff ping to pool \ node \ your proxy ?

**Comment by nekenet** [comment]
No, the connection is good with the pool. about 90 ms. but the hash-rate strongly swims from 1700 ± 500. if I connect the miners directly, then the hash on the pool shows 2000 ± 200. why did the speed drop so?

**Comment by 2010phenix** [comment]
try check not Hash rate but shares over time period 10 min \ 30 min
maybe another pool for test too  ± 500. big diff...

**Comment by nekenet** [comment]
does hash does not depend on the number of shares? about the share that's what the problem is:
stat of work 
          accepted sh       rj sh         total hs
pool 1      607             | 316 |     43328631
pool 2      432             | 128 |     113399641
and etc
what the problem?

**Comment by 2010phenix** [comment]
some misunderstanding i think...
check on pool directly with period XX min
check over this proxy with period XX min
and if shares count more like same... that losting hashrate is visual (500 h\s is big(25%) and you can see over test)
and do same on another pool that you know.. problem not a your pool
